### ISSUE TITLE
HttpServer and EventBase can support multi-thread?

### ISSUE THREAD

**Issue by xhzzhang** [yedf/handy#87]
Hello, I have a http API to process too many time task, then I need to process other API, so is it support multi-thread for http server?

**Comment by jovany-wang** [comment]
Yes. Please refer to this example https://github.com/yedf/handy/blob/master/examples/http-hello.cc#L12
